en/docs/extended_topics/how_to_convert_image_to_ofrecord.md

```diff
@@ -0,0 +1,147 @@
+# Convert the image files to OFRecord datasets
+
+In  [OFRecord Data Format](./ofrecord.md) and  [Loading and Preparing OFRecord Dataset](./how_to_make_ofdataset.md), We learned how to convert other dataset formats to OFRecord datasets separately and how to load OFRecord datasets.
```

Comment:
We learned how to ->  we learned how to 
convert other dataset formats to OFRecord datasets separately -> convert other dataset formats to OFRecord separately